python/vision_explanation_methods/common/constants.py

```diff
@@ -0,0 +1,15 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Defines constants for vision-explanation-methods."""
+
+from enum import Enum
+
+
+class Device(Enum):
+    """Specifies all possible device types."""
+
+    CPU = 'cpu'
```

Comment:
I wonder if this can be re-imported from https://github.com/microsoft/ml-wrappers/blob/d83115785f50aef4e627dc1eba92d9c042f7c11e/python/ml_wrappers/common/constants.py#L319 ?
Hmm, or perhaps not since this repo does not have a dependency on ml-wrappers.  Actually, it seems it does have a dependency here:
https://github.com/microsoft/vision-explanation-methods/blob/main/python/setup.py#LL44C6-L44C17
so perhaps indeed it might be easier to just re-import here and reduce duplication.

Reply:
good point!